operator/pkg/helmreconciler/reconciler.go

```diff
@@ -407,7 +407,6 @@ func (h *HelmReconciler) getCoreOwnerLabels() (map[string]string, error) {
 	if revision == "" {
 		revision = "default"
 	}
-	labels[label.IoIstioRev.Name] = revision
```

Comment:
can you explain this change?

Reply:
Noticed that because of this all charts in base get `istio.io/rev` which seems wrong, need to look more into it. Specifically having `istio.io/rev` for base validating webhook would cause new revision to patch it when it shouldn't.